### ISSUE TITLE
ISLE build fails on latest Docker Desktop

### ISSUE THREAD

**Issue by alxp** [Islandora/documentation#1892]
PR: https://github.com/Islandora-Devops/isle-dc/pull/183

Getting a build failure after re-downloading Docker Desktop for macOS and running make local.

```bash
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name 
1 error(s) decoding:

* error decoding 'Command': invalid command line string
make[1]: *** [docker-compose.yml] Error 15
make: *** [local] Error 2
```

Looking at the release notes for Docker Desktop, it looks like it now ships with a release candidate of Docker Compose 2.0, which has a new config file syntax.

https://docs.docker.com/compose/cli-command/

I've added a note in isle-dc's README.md with the workaround to still use the 1.x syntax: in the PR linked above.

**Comment by nigelgbanks** [comment]
Pull request to address the issue: https://github.com/Islandora-Devops/isle-dc/pull/198

**Comment by nigelgbanks** [comment]
Resolved by Islandora-Devops/isle-dc#198
